conftest.py

```diff
@@ -71,3 +74,11 @@ def pytest_ignore_collect(path, config):
             # If the current Python version does not meet the minimum required, skip this directory
             if sys.version_info[0:2] < min_required:
                 outcome.force_result(True)
+
+
+@pytest.mark.hookwrapper
+def pytest_runtest_teardown(item, nextitem):
+    for object in _CACHED_OBJECTS:
```

Comment:
might be worth adding an `invalidate_all_caches` helper to `ddtrace.utils.cache`?

That way you don't need to reference the private/global variable?

Reply:
I'm reluctant to introduce new internal code that looks like public API, so unless you feel very strongly about this helper I'd rather keep it like this 🙂 . Even better, if anybody knows how I could patch the decorators before they are applied when `pytest` is invoked, I could get rid of `_CACHED_OBJECTS` entirely.

Reply:
no strong opinion, just seemed like a potential place to make this easier to work with, but if this is literally the only place it'll ever exist, that should be fine.